### ISSUE TITLE
Catch errors in gulp tasks to keep watch task running

### ISSUE THREAD

**Issue by Mancherel** [zurb/foundation-sites-6#256]
It would be nice if any errors produced by the gulp tasks would be caught instead of thrown. This to keep gulp watch running. Now I have to restart the sequence every time I do a CSS typo, resulting in time wasted and yet another tab in chrome. I'm lazy, I know.

**Comment by zurbchris** [comment]
Is that in the testbed folder?

**Comment by Mancherel** [comment]
@zurbchris I downloaded the Advanced Project Template. Ran npm install and started the build process with npm start, all as stated in the documentation. When an error happens in the sass build (like when I do a typo or import a file that does not yet exist) the watch task will halt, kind of annoying in my opinion.
So I have done som researching and testing. The errors could be handeld by a package like gulp-plumber.

**This is how I added gulp-plumber**
* Add gulp-plumber locally to the project by running `npm install --save-dev gulp-plumber`
* Add gulp-plumber to gulpfile.js `var plumber = require('gulp-plumber');`
* Pipe plumber first in the sass chain and run sass in sync mode by calling sass.sync instead of sass directly (to enable sass to callback on error). 
```javascript
return gulp.src('./src/assets/scss/app.scss')
    .pipe(plumber())
    .pipe($.sass.sync({
      includePaths: paths.sass,
      outputStyle: (isProduction ? 'compressed' : 'nested'),
      errLogToConsole: true
    }))
    .pipe($.autoprefixer({
      browsers: ['last 2 versions', 'ie >= 9']
    }))
    .pipe(uncss)
    .pipe(gulp.dest('./dist/assets/css'));
```
* All good

<img width="408" alt="screen shot 2015-11-16 at 09 18 10" src="https://cloud.githubusercontent.com/assets/7570106/11177078/2bbbb19c-8c43-11e5-8324-2495d378075f.png">


Cheers
